index.html

```diff
@@ -0,0 +1,265 @@
+      <nav class="social-link">
+        <ul class="cont-list-author">
+          <li class="single-cont-author">
+            <a href="#" class="link-nav"><i class="fab fa-facebook-f"></i></a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav"><i class="fab fa-twitter"></i></a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav">English</a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav">Mypage</a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav">Logout</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="cont-navbar">
+        <div class="navbar-wrapper">
+          <a href="./index.html" class="link-page">
+            <figure class="logo-img-cont">
+              <img
+                src="./images/logo.png"
+                alt="logo"
+                class="logo-footer logo-navbar"
+              />
+              <div class="logo-text">
+                <h1 class="website-name logo-name">Earth</h1>
+                <h2 class="website-name logo-name">Planet</h2>
+              </div>
+            </figure>
+          </a>
+          <nav class="link-nav-cont">
+            <ul class="cont-list">
+              <li class="single-cont-list">
+                <a href="./about.html" class="link-nav-page">About</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#main" class="link-nav-page">Program</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#" class="link-nav-page">Join</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#" class="link-nav-page">Sponsor</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#" class="link-nav-page">News</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#" class="link-nav-page action-camp">CC Campaign</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+
+    <section class="header-cont">
+      <h1 class="heading">&nbsp;</h1>
+      <div class="intro">
+        <h4 class="intro-salutation">"Hello! Environmental protector"</h4>
+        <h1 class="intro-title">World Environment Summit 2021</h1>
+        <p class="intro-desc">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsum,
+          incidunt delectus optio illo ducimus vero recusandae minus doloremque
+          ex repellat deserunt ipsa cupiditate quos, dolore facere voluptas
+          facilis non reiciendis?
+        </p>
+        <h2 class="intro-date">2021.10.11(THU) ~ 16(FRI)</h2>
+        <h5 class="intro-location">
+          @ Lorem, ipsum dolor sit amet consectetur adipisicing.
+        </h5>
+      </div>
+    </section>
```

Comment:
- [ ] Please follow closely the design guidelines in the margin, padding, size, and color

Yours:
![Screenshot from 2021-10-15 19-17-35](https://user-images.githubusercontent.com/62705947/137528191-47a544bb-30ed-4fbc-abcb-f98c3917fb4b.png)
Original:
![Screenshot from 2021-10-15 19-17-39](https://user-images.githubusercontent.com/62705947/137528215-3fc45d44-f277-4a19-81ee-6df423367673.png)

Reply:
- [ ] You should follow the design guidelines given in the template. The yellow color you use is suits better with the theme that you choose for the project. :disappointed: However, We expect you to personalize only the content of the page, not the design itself. So as my previous code reviewer requested, I have to ask you to follow the design principles and change the color of these `yellow` parts to the original colors given in the design guidelines.